### PR TITLE
Update Gatsby and fix scroll issue

### PR DESCRIPTION
**Pull request by tobias-tengler**
A user reported the inability to scroll using arrow keys, with this change the website can be scrolled again, albeit the scrollbar is now partially hidden behind the header.

### CODE REVIEW

**Comment by tobias-tengler** [comment]
@rstaib Is it intentional that the .yarn directory with its cache was checked in?

File: website/src/components/misc/global-style.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ export const GlobalStyle = createGlobalStyle`
 
     --border-radius: 4px;
     --font-size: .833rem;
-  }
+  }
```
Comment (rstaib) [inline website/src/components/misc/global-style.tsx:15]: empty space

File: website/src/components/misc/global-style.tsx [hunk 2]
```diff
@@ -35,12 +35,10 @@ export const GlobalStyle = createGlobalStyle`
 
     > div:first-child {
       display: block;
-      height: 100vh;
```
Comment (rstaib) [inline website/src/components/misc/global-style.tsx:38]: removing height leads to another issue, where navigation does not take anymore the entire height of the content area.

File: website/src/components/misc/global-style.tsx [hunk 3]
```diff
@@ -35,12 +35,10 @@ export const GlobalStyle = createGlobalStyle`
...
 
       > div {
         display: flex;
         flex-direction: column;
-        height: 100vh;
```
Comment (rstaib) [inline website/src/components/misc/global-style.tsx:43]: same here

**Review by rstaib — CHANGES_REQUESTED** [review]
introduces another issue in regard to navigation height

**Pull request closed without merging**